### DISCUSSION TITLE
Zero-UI Blank page after login

### DISCUSSION THREAD

**Discussion by perunija** [dec0dOS/zero-ui#63] [Q&A]
Hello to the zerotier community :).
I have an issue to access the zerotier controller page. After login page I get blank screen. I've restarted the system but it didn't solve the issue.
Since I'm very new to this, I'm kindly asking for some guidance on how and where to collect proper logs in order to give you more details.
Cheers!

**Comment by dsm150** [comment]
I have same problem.

**Reply by perunija** [reply]
After doing some backward tracing I have discovered that one of the networks (the last I've created) was causing the issue as it had wrong managed route settings that have created a "loop".   

So, I was investigating by myself and inside the zerotier-one Docker container I was able to locate the network json file of the created network that had created the issue. The file (16-digit-number.json) is located in the following path 

"/var/lib/zerotier-one/controller.d/network/file.json" 

Also there's a folder with the same network ID. I've decided to remove both with 

`rm -rf /var/lib/zerotier-one/controller.d/network/file.json` 

and 

`rm -rf /var/lib/zerotier-one/controller.d/network/folder` 

After making those changes I've exited the zerotier-one Docker container and then I was instantly able to access the Zero-UI via web browser.
This might help someone to resolve the issue.

**Reply by dsm150** [reply]
Unfortunately, it didn't help me. I deleted all "file.json" and "folder", rebooted the VM, but I still see a blank screen after logon.

**Reply by Ashitaka57** [reply]
Maybe related to the fact that you are using Edge instead of Chrome or Firefox.

**Comment by youtehub** [comment]
maby Clean the Edge cache files and data, it  will work normally.OR Try opening it in inPrivate mode
